nubia/internal/parser.py

```diff
@@ -23,7 +23,9 @@ def _bool_transform(x):
 
 
 def _str_transform(x):
-    return x.strip("\"'")
+    if x and ((x[0] == x[-1] == '"') or (x[0] == x[-1] == '"')):
```

Comment:
Second and third conditions are the same, should be
```suggestion
    if x and ((x[0] == x[-1] == '"') or (x[0] == x[-1] == "'")):
```